posts/views.py

```diff
@@ -1,3 +1,15 @@
-from django.shortcuts import render
+from django.shortcuts import redirect
+from posts.models import Post
+from django.views import generic
 
-# Create your views here.
+
+def PostCreateView(request):
+    if (len(request.POST['content']) <= 256):
+        post = Post(content=request.POST['content'], author=request.user)
+        post.save()
+    return redirect(request.META['HTTP_REFERER'] or 'landingpage')
```

Comment:
You should use django messages to show the user, that something went wrong.
https://docs.djangoproject.com/en/1.8/ref/contrib/messages/#adding-a-message